packages/lending/src/protocols/openocean-v2/configs.ts

```diff
@@ -1 +1,5 @@
+import * as logics from '@protocolink/logics';
+
 export const NAME = 'openocean-v2';
```

Comment:
Would it be better if we use `ID` instead of `NAME`?

Reply:
Agree and changed.